src/backend/db/products.js

```diff
@@ -8,23 +8,257 @@ import { v4 as uuid } from "uuid";
+    sellingPrice:"999",
+    discountOffer:"40% off",
+    IsSoldOut:true,
+    fastDelivery:false,
+    image:".assets/MEN/Acqua-di-Gio-Essenza-Giorgio-Armani_men.jpg",
+    categoryName: "Men",
+  },
+  {
+   _id: uuid(),
+    productName:"Allure-Homme-Sport-Chanel - 100ml (MEN)",
+    qunatity:"100 ml",
+    brand:"Chanel",
+    rating:"3.5",
+    IsRating:false,
+    originalPrice:"1500",
+    sellingPrice:"1000",
+    discountOffer:"20% off",
+    IsSoldOut:false,
+    fastDelivery:true,
+    image:".assets/MEN/Allure-Homme-Sport_chanel-men.jpg",
+    categoryName: "Men",
+  },
+  {
+    _id: uuid(),
+    productName:"Armani-prive-cuir- 100ml (MEN)",
+    qunatity:"100 ml",
+    brand:"Armani",
+    rating:"3.8",
+    IsRating:false,
+    originalPrice:"1800",
+    sellingPrice:"1200",
+    discountOffer:"30% off",
+    IsSoldOut:false,
+    fastDelivery:true,
+    image:".assets/MEN/armani-prive-cuir_men.jpg",
+    categoryName: "Men",
+  },
+  {
+    _id: uuid(),
+    productName:"Armani-Royal_Oud- 150ml (MEN)",
+    qunatity:"150 ml",
+    brand:"Armani",
+    rating:"4",
+    IsRating:true,
+    originalPrice:"3000",
+    sellingPrice:"2000",
+    discountOffer:"15% off",
+    IsSoldOut:false,
+    fastDelivery:true,
+    image:".assets/MEN/Armani-Royal-Oud_men.jpg",
+    categoryName: "Men",
+  },
+  {
+    _id: uuid(),
+    productName:"Armani-sundigioia- 150ml (MEN)",
+    qunatity:"150 ml",
+    brand:"Armani",
+    rating:"4.5",
+    IsRating:true,
+    originalPrice:"4000",
+    sellingPrice:"2000",
+    discountOffer:"50% off",
+    IsSoldOut:false,
+    fastDelivery:false,
+    image:".assets/MEN/armanisundigioia_men.jpg",
+    categoryName: "Men",
+  },
+  {
+    _id: uuid(),
+    productName:"Chanel-paris-1957-150ml (MEN)",
+    qunatity:"150 ml",
+    brand:"Chanel",
+    rating:"4.6",
+    IsRating:true,
+    originalPrice:"13000",
+    sellingPrice:"10000",
+    discountOffer:"20% off",
+    IsSoldOut:true,
+    fastDelivery:false,
+    image:".assets/MEN/chanel-paris-men.jpg",
+    categoryName: "Men",
+  },
+  {
+    _id: uuid(),
+    productName:"Chanel-platinum-egoiste- 150ml (MEN)",
+    qunatity:"150 ml",
+    brand:"Chanel",
+    rating:"4.8",
+    IsRating:true,
+    originalPrice:"3000",
+    sellingPrice:"1799",
+    discountOffer:"30% off",
+    IsSoldOut:false,
+    fastDelivery:true,
+    image:".assets/MEN/chanel-platinum-egoiste_men.jpg",
+    categoryName: "Men",
+  },
+  {
+    _id: uuid(),
+    productName:"Gucci-IntenseOud-UseMe-100ml (MEN)",
+    qunatity:"100 ml",
+    brand:"Gucci",
+    rating:"4.9",
+    IsRating:true,
+    originalPrice:"3000",
+    sellingPrice:"899",
+    discountOffer:"60% off",
+    IsSoldOut:true,
+    fastDelivery:false,
+    image:".assets/MEN/Gucci-IntenseOud-UseMe_men.png",
+    categoryName: "Men",
+  },
+  {
+    _id: uuid(),
+    productName:"Gucci-oriental-base- 150ml (MEN)",
+    qunatity:"150 ml",
+    brand:"Gucci",
+    rating:"3.9",
+    IsRating:true,
```

Comment:
naming convention is different for IsRating and IsSoldOut and all other keys are different convention. please look into this.